core/contract/src/main/java/org/eclipse/dataspaceconnector/contract/negotiation/ConsumerContractNegotiationManagerImpl.java

```diff
@@ -571,9 +580,9 @@ public ConsumerContractNegotiationManagerImpl build() {
             Objects.requireNonNull(manager.commandQueue, "commandQueue");
             Objects.requireNonNull(manager.commandRunner, "commandRunner");
             Objects.requireNonNull(manager.observable, "observable");
-    
+
```

Comment:
There is actually a default implementation set in the Builder:
```
private Builder() {
            manager = new ConsumerContractNegotiationManagerImpl();
            manager.telemetry = new Telemetry(OpenTelemetry.noop()); // default noop implementation
        }
```

But since users can still override it with null, I will add the check too. Thanks!